interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: `this` is used to access informatin within a class. 
 
-  Researched answer:
+  Researched answer: `this` is  a javascript keyword that refers to the object it belongs to.  
 
 
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: A) Yarn is a spun thread used for knitting, weaving, or sewing. In Javascript, yarn is part of the Jest testing framework. B) I'm not sure about this one.
 
-  Researched answer:
+  Researched answer: A) Yarn is a file manager for the Jest testing framework. B) This was hard to find but I believe the yarn.lock file is modified when we run yarn. This file holds the dependencies used for the project.
```

Comment:
Yarn is a package manager developed by facebook when they found that npm (another package manager) was too difficult yo refactor for their purposes. It can manage the package Jest and many others!